### DISCUSSION TITLE
Install kimai on Docker - Raspberry Pi

### DISCUSSION THREAD

**Discussion by safibet** [tobybatch/kimai2#564] [General]
Hi,

Is it possible to install Kimai on Docker on my Raspberry Pi 5?

I tried to follow some tutorials but I get errors.

Thanks

**Comment by tobybatch** [comment]
Maybe you could share those errors

**Comment by safibet** [comment]
sure: WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
